jobs/scheduled/antivirus_health_check.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Jobs
+  class AntivirusHealthCheck < ::Jobs::Scheduled
+    every 1.minute
```

Comment:
I don't think we should be relying on Sidekiq here to push the metric to the prometheus client. Instead, we should introduce an API so that custom collectors can be added to the global collector such that collection of metrics can happen as part of this thread: https://github.com/discourse/discourse-prometheus/blob/230aa980599841aff2378f12a41f587db5241823/lib/reporter/global.rb#L31-L36